third_party/move/move-compiler-v2/tests/lints/model_ast_lints/blocks_in_conditions_warn.exp

```diff
@@ -0,0 +1,55 @@
+
+Diagnostics:
+warning: Having blocks in conditions make code harder to read. Consider rewriting this code.
```

Comment:
Probably should be `warning: [lint] <message>`

Reply:
done.